index.js

```diff
@@ -40,9 +40,9 @@ process.env.HOME = options.tempdir; // Workaround: Bower ends up using $HOME/.lo
 buildService(options)
 	.listen()
 	.then(() => {
-		return primeNpmCache(options.npmRegistryURL);
+		return primeNpmCache(options.npmRegistryURL).catch(console.log);
```

Comment:
Let's use console.error so that it goes to stderr instead 👍 
```suggestion
		return primeNpmCache(options.npmRegistryURL).catch(console.error);
```